.github/workflows/wasm32-unknown-unknown-macOS.yml

```diff
@@ -11,15 +11,15 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2020-05-13
+          toolchain: nightly-2021-01-29
           override: true
       - name: Cache Cargo
         uses: actions/cache@v2
         with:
           path: |
             ~/.cargo/registry
             ~/.cargo/git
-          key: ${{ github.workflow }}-${{ github.job }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          key: ${{ github.workflow }}-${{ github.job }}-toolchain-nightly-2021-01-29-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
You can replace the hardcoding of the toolchain name in the key with `${{ env.INPUT_TOOLCHAIN }}`